app/models/concerns/oid_generator.rb

```diff
@@ -0,0 +1,65 @@
+module OidGenerator
+  extend ActiveSupport::Concern
+
+  if Rails.application.config.oid_prefix.blank?
+    raise 'Missing OID prefix, cannot continue. Configure this prefix in config/initializers/oid_prefix.rb'
+  end
+  CLASS_PREFIXES = {  Form: "#{Rails.application.config.oid_prefix}.1",
+                      Question: "#{Rails.application.config.oid_prefix}.2",
+                      ResponseSet: "#{Rails.application.config.oid_prefix}.3" }.freeze
```

Comment:
Should probably just make these individually configurable instead of generating the prefix for each class.  That way the prefixes can be updated if needed on a class basis without the need of modifying code to deal with it.